lib/fs/fs.ml

```diff
@@ -45,12 +52,14 @@ let rec to_tree path =
            (Sys.readdir path))
     in
     let dirname = Filename.basename path in
-    Dir (dirname, children)
+    Dir { name = dirname; children }
```

Comment:
Let's just rename `dirname` above to `name` to leverage punning and write just `Dir { name; children }`.

Reply:
In fact, let's do one more step and move this new `name` before `if`. 
This way we can reuse it in both `Dir` and `File` cases.